nginx.conf.d/api_redirect.conf

```diff
@@ -0,0 +1,3 @@
+location ~ ^/api(.*) {
+    return 301 $scheme://api.brasil.io$1;
+}
```

Comment:
Esse arquivo precisa ser colocado manualmente no `/home/dokku/<app-name>/nginx.conf.d`, certo?
Acho que seria legal alguma alternativa que não dependesse de copiar manualmente pra evitar erros. Eu já usei o [`nginx.conf.sigil`](http://dokku.viewdocs.io/dokku/configuration/nginx/#customizing-the-nginx-configuration) em um projeto.